packages/hint-validate-summary/README.md

```diff
@@ -0,0 +1,88 @@
+# validate-summary (`@hint/hint-validate-summary`)
+
+Summary tag should have display: list-item only.
+
+## Why is this important?
+
+Changing the display on a summary tag to anything other than list-item
+will cause the open/closeicon to disappear in browsers which implement
+the latest version of the standard.(eg: Firefox).
+Chrome has
+<a href="https://bugs.chromium.org/p/chromium/issues/detail?id=590014&q=component%3ABlink%3EHTML%3EDetails%20&colspec=ID%20Pri%20M%20Stars%20ReleaseBlock%20Component%20Status%20Owner%20Summary%20OS%20Modified">
+a bug open to track aligning with the spec and Firefox in the future.</a>
+
+## What does the hint check?
+
+The hint examines any rule that target a `summary` element and change the
+display to anything other than `list-item`.
+
+### Examples that **trigger** the hint
+
+`Any summary tag that alters the display to anything other than list-item`
+
+Example:
+
+```summary {
+    display: block;
+}```
+
+`Class or id of summary tag that alters the display property`
+
+Example:
+
+```summary.foo {
+    display: inline;
+}```
+
+### Examples that **pass** the hint
+
+`If the summary tag does not change the display property at all`
+
+Example:
+
+```summary {
+    margin: 10px;
+}```
+
+`If there is no summary tag in the css file`
+
+## Can the hint be configured?
+
+If this hint allows some configuration, please put the format and
+options for the user.
```

Comment:
Also, we can just remove this section since we're not currently allowing this hint to be configured.